roadmap/implementers-guide/src/runtime/configuration.md

```diff
@@ -35,9 +35,6 @@ fn update_configuration(f: impl FnOnce(&mut HostConfiguration)) {
     *pending = Some(x);
   })
 }
-
-/// Get the GlobalValidationData, assuming the context is the parent block.
-fn global_validation_data() -> GlobalValidationData;
```

Comment:
`local_validation_data` was queried from `paras.md`. `global_validation_data` was declared here, in `configuration.md`.

In the presented changes, `persisted_validation_data` is queried from `paras.md` but where do I query `full_validation_data` to implement the runtime API call with the same name?

Reply:
The runtime API will need to draw this information out from many different modules. It wasn't obvious where its home should be.

Reply:
yeah I actually was triggered by that: it wasn't for me where would be the home and then I thought that the chances are it might be not obvious for the implementer either.

but that's fine, we can figure out this later